test/composites/alert/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const assert = require('chai').assert;
+const Classlist = require('classlist');
```

Comment:
I need to clarify the support requirements for dqpl...if its IE10+, then we're good (unless we want to utilize the 2nd param of toggle (which we don't)).  Once I figure out what we need to support I can strip this module out and replace calls to it using the native browser method